docs/dev/releasing.rst

```diff
@@ -0,0 +1,500 @@
+GitHub and create the GitHub Release object. In order to do this, it is required
+to have a GitHub API token that can be used to create and modify the releases
+for the repository.
+
+To get an access token, perform the following:
+
+1. Open the `Settings > Personal access tokens`__ page on GitHub.
+2. Press the *Generate new token* dropdown.
+
+   1. Select a "general use"/\ "classic" token. (Creating a fine-grained access
+      token requires administrative approval before it can be used.)
+
+3. Set a *note* for the token that explains its purpose. This can be arbitrary,
+   but is useful when reviewing the token later.
+4. Set the expiration to the minimum (we only need the token for the duration of
+   this release).
+5. In the scopes, enable the ``public_repo`` scope.
+6. Generate the new token. Be sure to copy the access token a save it for later,
+   as it won't be recoverable once the page is unloaded.
+
+__ https://github.com/settings/tokens
+
+.. XXX: The following applies to fine-grained access tokens. Not sure if these work yet?
+
+   1. Open the `Settings > Personal access tokens`__ page on GitHub.
+   2. Press the *Generate new token* dropdown.
+
+      1. Select a "Find-grained, repo-scoped" token. The general use token is also
+         acceptable but is very coarse and not as restricted.
+
+   3. Set a token name. This can be arbitrary, but would be best to refer to the
+      purpose so that it can be recognized later.
+   4. Set the expiration to the minimum (we only need the token for the duration of
+      this release).
+   5. Set the *Resource owner* to **mongodb** (**mongodb** refers to the GitHub
+      organization that owns the repository that will contain the release. A
+      personal account resource owner will only have access to the personal
+      repositories.)
+   6. Under *Repository access* choose "Only select repositories".
+   7. In the repository selection dropdown, select ``mongodb/mongo-c-driver``.
+   8. Under *Permissions > Repository permissions*, set the access level on
+      *Contents* to *Read and write*. This will allow creating releases and
+      publishing release artifacts. No other permissions need to be modified.
+      (Selecting this permission may also enable the *Metadata* permission; this is
+      normal.)
+
+
+Do the Release
+##############
+
+.. highlight:: console
+.. default-role:: bash
+
+The release process at this point is semi-automated by scripts stored in a
+separate repository.
+
+.. hint::
+
+   It may be useful (but is not required) perform the following steps within a
+   new Python `virtual environment`__ dedicated to the process.
+
+__ https://docs.python.org/3/library/venv.html
+
+
+.. _do.stopwatch:
+
+Start a Release Stopwatch
+*************************
+
+Start a stopwatch before proceeding.
+
+
+Clone the Driver Tools
+**********************
+
+Clone the driver tools repository into a new directory, the path to which will be
+called `$DRIVER_TOOLS`::
```

Comment:
```suggestion
called `$CDRIVER_TOOLS`::
```

Suggested rename to avoid confusing with `DRIVERS_TOOLS` use in other scripts referring to the mongodb/drivers-evergreen-tools repo.